src/schemas/qualifications.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose')
+const uniqueValidator = require('mongoose-unique-validator')
+
+const qualificationsSchema = new mongoose.Schema({
+  title: {
+    type: String,
+    required: true
+  },
+  type: {
+    type: String,
+    required: true
+  }
+})
+
```

Comment:
This is incorrectly implemented. If a qualification relates to a user, a `user` field of type `objectID` should be added to the `qualification` schema.

Reply:
it is not related to user

Reply:
How so? You've declared a relationship here:
1. https://github.com/kyleag/marketplays-api/pull/51/files#diff-8ad680d86218600b170923b760b1c886d9321c348d638ee4b6e8de3bdc77b2c8R6 - User relating to qualification aliased as `skill`

2. https://github.com/kyleag/marketplays-api/pull/51/files#diff-0801755b6eeee3b1698911aa7f2a3a49202e4f43d5634eabbb73ed99c2aeaa45R6 - User relating to qualification aliased as `knowledge`

Are you sure you're aware of what you're doing? I'm starting to believe that you don't.